Export API to v1.0 (#21700)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/security-caseoperation-get.md

    }
}
```

> **Note:** In case of export case operation, you will  receive the downloadURL, fileName and size in the exportfileMetadata property of the response. The actual operation where you will initiate the download is an operation managed on your side. You may choose to download it to your local machine or export it to your storage account.

Reviewer: ```suggestion
**Note:** In the case of the export case operation, you will receive the downloadURL, fileName, and size in the exportfileMetadata property of the response. The operation where you will initiate the download is managed on your side. You may download it to your local machine or export it to your storage account.
```

> In order to avoid the interactive sign-in page and fully automate the process, please follow the following steps

Reviewer: ```suggestion
To avoid the interactive sign-in page and fully automate the process, please follow the following steps:
```

> 1. Provision the MicrosoftPurviewEDiscovery application by leveraging [Create ServicePrincpal](../api/serviceprincipal-post-serviceprincipals.md) for the application id b26e684c-5068-4120-a679-64a5d2c909d9

Reviewer: ```suggestion
1. Provision the Microsoft Purview eDiscovery application by leveraging [Create ServicePrincpal](../api/serviceprincipal-post-serviceprincipals.md) for the application id b26e684c-5068-4120-a679-64a5d2c909d9
```

> 2. After provisioning, please raise request for user delegated permission of ediscovery.Download.Read for the above application from your third party application which you would be using for the graph. 

Reviewer: ```suggestion
2.Once the application is provisioned, request user-delegated permissions for discovery.Download.Read from the tenant admin. This request should be made from your third-party application interacting with Microsoft Graph.
```

> 3. Ensure the tenant admin approve the request

Reviewer: ```suggestion
3. Make sure the tenant admin approves the request
```

> 4. Now, you can add the scope for the application to your existing scope in the script and ensure the headers allow ("X-AllowWithAADToken", "true"); 

Reviewer: ```suggestion
4. Add the scope for the application to your existing script and make sure the headers include ("X-AllowWithAADToken", "true");. 
```

> With these steps, user authorisation will take place in the automated manner and you will not receive manual interactive sign in page. 

Reviewer: ```suggestion
By following these steps, the user authorization process will be automated, and you won't encounter a manual interactive sign-in page.
```
